src/FixerFactory.php

```diff
@@ -259,6 +259,7 @@ private function getFixersConflicts(FixerInterface $fixer)
     {
         static $conflictMap = array(
             'no_blank_lines_before_namespace' => array('single_blank_line_before_namespace'),
+            'no_short_echo_tag' => array('opening_tag_plus_echo_to_short_echo_tag'),
```

Comment:
I would rather make one configurable fixer.
Like `short_echo_tag` with configuration which version to use.